client/verta/tests/test_permissions/test_workspace.py

```diff
@@ -61,47 +61,7 @@ def test_model_ver(self, client, organization, created_entities):
         )
 
         # organization workspace
-        reg_model = client.create_registered_model(workspace=organization.name)
+        reg_model = client.create_registered_model(workspace=workspace.name)
         created_entities.append(reg_model)
-        assert reg_model.create_version().workspace == organization.name
+        assert reg_model.create_version().workspace == workspace.name
 
-
-class TestClientGetWorkspace:
```

Comment:
fyi, this test is very important. We should add it again when workspace creation is implemented.

Reply:
reverted removal. I think it can be added here